.github/workflows/deploy-pnpm.yaml

```diff
@@ -0,0 +1,48 @@
+# Workflow intended to properly build the GitHub Pages Site. Using PNPM 
+
+name: Deployment Docs - PNPM 
+
+on:
+  # Trigger deployment on every push to main branch.
+  push:
+    branches: [main]
+  # Trigger deployment manually.
+  workflow_dispatch:
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest 
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Fetch all commits to get last updated time or other git log info 
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+      - name: Setup PNPM 
```

Comment:
The pnpm project offers an action to setup pnpm. I don't really care whether we use it or not, but wanted to mention it.

Example usage:

https://github.com/pnpm/action-setup#use-cache-to-reduce-installation-time